runtime/doc/options.txt

```diff
@@ -6552,7 +6539,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 'ttybuiltin' 'tbi'	Removed. |vim-differences| {Nvim}
 
 				     *'ttyfast'* *'tf'* *'nottyfast'* *'notf'*
-'ttyfast' 'tf'		Removed. |vim-differences| {Nvim}
+'ttyfast' 'tf'		Deprecated. |vim-differences| {Nvim}
+			'set ttyfast' is ignored, 'set nottyfast' is an error.
```

Comment:
OK, I just included it here because the documentation didn't reflect the status of the setting.